hseling-web-cat-and-kittens/hseling_web_cat_and_kittens/templates/base.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel='shortcut icon' type='image/x-icon' href="{{ url_for('static', filename='img/favicon.ico') }}" />  
+
+    <title>{% block title %}CAT&Kittens{% endblock %}</title>
+
+    {% block links %}{% endblock %}
+</head>
+
+<body>
+    <header class="masthead mb-auto">
+        <div class="inner">
+            <nav class="navbar navbar-expand-md navbar-light fixed-top bg-light" style="text-align: left !important;">
+                <a class="navbar-brand" href="{{ url_for('index') }}">CAT&Kittens</a>
```

Comment:
All special symbols within HTML should be replaced by their counterparts: "& -> \&amp;"